test/tests/inittest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This have to be first test for cockpit
+#  install build dependencies
+#  download new version of cockpit from git 
+#  compile it
+# it is basic test compilation must PASS
+
+set -x
+
+BUILD="build1"
+PACKAGE="cockpit"
+SOURCE="https://github.com/$PACKAGE-project/$PACKAGE.git"
+
+#
+yum-builddep -y $PACKAGE
```

Comment:
Yes, it is little bit problem (I had to add PCP* dependencies to another yum command)
Is it able to find actual dependencies ?